test/E2E/Pay_Mint_Redeem_E2E.t.sol

```diff
@@ -265,11 +265,13 @@ contract Test_TiersHook_E2E is TestBaseWorkflow {
         assertEq(IJB721TiersHook(dataHook).payCreditsOf(beneficiary), valueSent);
     }
 
-    // TODO: This needs care (fuzz fails with insuf reserve for val=10)
```

Comment:
tier prices start are tier*10, 9 should fails but 10 passed for me (hence the boundaries)